manager.go

```diff
@@ -199,7 +201,9 @@ func listChangedFiles(dir string, gitBaseRef string) ([]string, error) {
 	}
 
 	if len(untracked) > 0 {
-		return nil, fmt.Errorf("repository has untracked files: %v", untracked)
+		log.Fatal().
```

Comment:
As mentioned here: https://github.com/mineiros-io/terramate/pull/141
Calling Fatal on the middle of some computation in Go is usually not a good idea, specially because any added defers will not be executed on this case (and they could do important cleanup).